Library/Homebrew/attestation.rb

```diff
@@ -121,7 +119,15 @@ def self.check_core_attestation(bottle)
         url_sha256 = Digest::SHA256.hexdigest(bottle.url)
         subject = "#{url_sha256}--#{bottle.filename}"
 
-        backfill_attestation = check_attestation bottle, BACKFILL_REPO, BACKFILL_REPO_CI_URI, subject
+        # We don't pass in a signing worfklow for backfill signatures because
+        # some backfilled bottle signatures were signed from a branch, and others
```

Comment:
Nitpick: let's be explicit here and say which branch.

Reply:
And which repo

Reply:
Done.